submissions.html

```diff
@@ -0,0 +1,149 @@
+      The manuscript should not exceed 5 pages, and it should include:
+      <ul>
+        <li>Title of the work, author(s) name(s), and affiliation(s) (if applicable).</li>
+        <li>Abstract (max 200 words).</li>
+        <li>Artistic concept and description of the work.</li>
+        <li>Relevant technical details of the work.</li>
+        <li><em><strong>A Clear description of the conceptual and/or technical relationship between the work and quantum computation.</strong></em>Please note that quantum-inspired works are <strong>not direct synonyms</strong> of quantum computer music nor quantum computation-inspired work. We expect works that include quantum computation either technically, conceptually, or aesthetically.</li>
+        <li>In case of a live performance, include a tech rider, specifying your minimum requirements and the equipment you will bring.</li>
+        <li>Links to audio/video examples of the work (if available).</li>
+        <li>Include references to any relevant supporting literature whenever possible.</li>
+     </ul>
+
+     All manuscripts will be reviewed for originality, academic standards, appropriateness of topic, readability (language), and technical feasibility. The audio/video file will be assessed for its artistic/conceptual quality and relevance to the manuscript and the topic.<br><br> 
+     Please note that currently, we are able to provide only minimal technical equipment for the live performances. We will provide a stereo sound system, a projector, and potentially internet connection. Special situations can be discussed, but you should consider bringing your own equipment and musical instruments, whenever applicable. Please make sure to include this information in your tech rider.<br><br>
+
+</ol>
+
+
+<p class='font-medium'>Paper Submission</p>
+<!-- #E40066 color for when submissions are closed. -->
+<p>The paper submission for ISQCMC '25 is currently <a href="mailto:quantummusicsubmission@gmail.com"><span style='color: #03CEA4;'>open</span></a>. Please refer to the detailed instructions below to ensure a smooth peer-review and avoid unnecessary rejections of your submission.</p>
+
+
+The deadline for submission is <strong>May 09, 2025</strong>. <br>
+
+<p>Paper Submission Instructions</p>
+Please make sure the paper is formatted according to the <a href="https://www.overleaf.com/latex/templates/isqcmc-2025-template/qyfngrjtyvjv"><u>ISQCMC 2025 Template</u></a>, is <em> anonymized</em> (see the Anonymization section below),and does not exceed 5 pages.<br><br>
+
+All submissions are to be sent to <a href="mailto:quantummusicsubmission@gmail.com">quantummusicsubmission@gmail.com</a>, and <strong> should contain the following information and files:</strong>
```

Comment:
please avoid having the email literally on the website to avoid spam web scrappers. as an example refer to what i did on https://github.com/isqcmc/isqcmc2025/blob/main/index.html

Reply:
clever!

Reply:
![](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExbWc2cms2NngwbXZ4OWs2dHk5eHJzNG9sMXc5MG1rNzdueGttcTA3cSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/d3mlE7uhX8KFgEmY/giphy.gif)